bool mission_local

float64 waypoint_x
float64 waypoint_y
float64 waypoint_longitude
float64 waypoint_latitude

float64 timeout
---
bool result
